### ISSUE TITLE
Feature: reverse lookup element to vm

### ISSUE THREAD

**Issue by simplesmiler** [vuejs/vue-devtools#105]
It's possible to "Inspect DOM" for given vm, and Chrome will open "Elements" panel and select the root element of vm.

It would be helpful, if there was a way to do the reverse of that. Given the element, open "Vue Devtools" panel and select the vm this element is governed by (if any).

**Comment by simplesmiler** [comment]
MIght be slightly complicated for slots, because for slots the closest element with viewmodel bound to it is not the viewmodel the data comes from.

``` js
var Home = {
  template: `
    <div>
      <tab-group>
        <tab-page v-for="tab in tabs">
          <div>{{ <belongs to "home" vm, not to "tab-page" vm> }}</div>
        </tab-page>
      </tab-group>
    </div>
  `,
};
```

**Comment by blake-newman** [comment]
@simplesmiler Any luck with this... I don't see that there is many other tools with this capability.

**Comment by simplesmiler** [comment]
I think the best way to do that would be to inject a vm reference to every element this vm governs. It's already done with root elements (`element.__vue__`).

Don't have time to try to implement it yet.

**Comment by simplesmiler** [comment]
Another complication is that in Vue 2 element may belong to multiple instances. The example below is producing a fragment (elementless) instance in Vue 1, but in Vue 2 it is absolutely valid. The `__vue__` seems to point at the parent instance.

Example: https://jsfiddle.net/simplesmiler/5LydL0hz/

**Comment by michalsnik** [comment]
I'm not sure it's worth the hassle actually. I don't see any interest around this functionality since it was proposed, so I'm going to close it now, and we'll rather focus on more vital things :)

**Comment by Akryum** [comment]
If I understand OP correctly, it is implemented by #479
